docs/getting-started/installing-with-docker.md

```diff
@@ -58,9 +60,9 @@ Then, start the container:
 ```bash
 docker run \
   --name resoto \
-  -e RESOTOWORKER_AWS_ACCESS_KEY_ID=YOUR_ACCESS_KEY_ID \
-  -e RESOTOWORKER_AWS_SECRET_ACCESS_KEY=YOUR_ACCESS_KEY \
-  -e RESOTOWORKER_COLLECT='aws example' \
+  -e RESOTOWORKER_OVERRIDE0 collect=aws,example
+  -e RESOTOWORKER_OVERRIDE1 aws.access_key_id=YOUR_ACCESS_KEY_ID \
+  -e RESOTOWORKER_OVERRIDE2 aws.secret_access_key=YOUR_SECRET_ACCESS_KEY \
```

Comment:
I feel like this should be removed, and instead a section added explaining how to use the new `configs` command. What happens if there is an override and a user tries to use the `configs` command later to make changes?

Reply:
Well for now I just wanted to get the docs into a working state again. Making it pretty is step 2

Reply:
Sure, but would it be better to just omit these?  It's not required to "get started" and get Resoto up and running.  We can maybe just add a link to the `configs` command CLI reference for now.

Reply:
If you omit them then nothing is being collected?